### ISSUE TITLE
Screen goes into sleep mode

### ISSUE THREAD

**Issue by Cwpute** [agateau/burgerparty#14]
Not a serious issue considering the speed of this game, but the screen goes into sleep mode if it is let without input.

The standard for fullscreen games is usually not to allow sleep mode.

**Comment by agateau** [comment]
I am not sure this is a problem: isn't it a good thing to save battery this way?

**Comment by HarriBuh** [comment]
> I am not sure this is a problem: isn't it a good thing to save battery this way?

No, it isn't. It's both annoying and distracting, and the phone should prioritize the game in the foreground.
Battery saving while playing a game is nonesense.

**Comment by HarriBuh** [comment]
We are talking about the phone getting idle WHILE playing your game. That's a different story. And I don't remember a single game ever going into dozing mode while being active. That's plain stupid.

**Comment by agateau** [comment]
> We are talking about the phone getting idle WHILE playing your game. That's a different story.

Are we?

Let me quote the issue description: 

> the screen goes into sleep mode if it is let without input.

What you are describing is a different problem, and a valid one. What's your hardware?

**Comment by HarriBuh** [comment]
Yes, screen goes off when not doing any inputs, BUT still while your app is ACTIVE.
I'm always on Google's  "extreme" battery mode, which should only affect idling/ background apps.

Google Pixel 9
